crypto/merklesignature/merkleSignatureScheme.go

```diff
@@ -78,6 +78,14 @@ type (
 	// Verifier is used to verify a merklesignature.Signature produced by merklesignature.Secrets.
 	// It validates a merklesignature.Signature by validating the commitment on the GenericVerifyingKey and validating the signature with that key
 	Verifier [MerkleSignatureSchemeRootSize]byte
+
+	//KeyRound represents an ephemeral signing key with it's corresponding round
+	KeyRound struct {
+		_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+		Round               uint64                    `codec:"rnd"`
+		EphemeralSigningKey *crypto.GenericSigningKey `codec:"key"`
```

Comment:
Could have just used `Key` instead, it seems a bit verbose considering what it's used for